storage/mysql/mysql.go

```diff
@@ -118,7 +118,13 @@ func (s *Storage) ReadCheckpoint(ctx context.Context) ([]byte, error) {
 	}
 
 	var checkpoint []byte
-	return checkpoint, row.Scan(&checkpoint)
+	if err := row.Scan(&checkpoint); err != nil {
+		if err == sql.ErrNoRows {
+			return nil, nil
```

Comment:
Please document this behaviour in the method comment if this is part of the contract (i.e. document that no checkpoint will not return an error, but will return an empty byte slice).

Reply:
Added method comments to `ReadCheckpoint`, `ReadTile` and `ReadEntryBundle`.